src/jit/codegenxarch.cpp

```diff
@@ -773,8 +773,8 @@ void CodeGen::genCodeForBinary(GenTree* treeNode)
     GenTreePtr op1 = treeNode->gtGetOp1();
     GenTreePtr op2 = treeNode->gtGetOp2();
 
-    // Commutative operations can mark op1 as contained to generate "op reg, memop/immed"
-    if (op1->isContained())
+    // Commutative operations can mark op1 as contained or used from spill to generate "op reg, memop/immed"
```

Comment:
>// Commutative operations can mark op1 as contained or used from spill to generate "op reg, memop/immed [](start = 4, length = 103)

In general, given containment isn't used to check whether an operand is used from reg or memory by codegen routines, it would be a good idea to update comments in relevant places not to use the word "contained".